app/src/test/java/org/simple/clinic/enterotp/EnterOtpScreenControllerTest.kt

```diff
@@ -113,57 +111,45 @@ class EnterOtpScreenControllerTest {
   }
 
   @Test
-  @Parameters(
-      "|true",
-      "1|true",
-      "11|true",
-      "111|true",
-      "1111|true",
-      "11111|true",
-      "111111|false"
-  )
-  fun `when an otp lesser than the required length is submitted, an error must be shown`(
-      otp: String,
-      shouldShowError: Boolean
-  ) {
+  fun `when an otp of the right length is submitted, an error must not be shown`() {
     whenever(ongoingLoginEntryRepository.entryImmediate()).doReturn(ongoingLoginEntry)
     whenever(loginUserWithOtp.loginWithOtp(phoneNumber, pin, otp)).doReturn(Single.just<LoginResult>(Success))
 
     uiEvents.onNext(EnterOtpSubmitted(otp))
 
-    if (shouldShowError) {
-      verifyZeroInteractions(loginUserWithOtp)
-      verify(screen).showIncorrectOtpError()
-    } else {
-      verify(screen, never()).showIncorrectOtpError()
-    }
+    verify(screen, never()).showIncorrectOtpError()
   }
 
   @Test
-  @Parameters(
-      "|false",
-      "1|false",
-      "11|false",
-      "111|false",
-      "1111|false",
-      "11111|false",
-      "111111|true"
```

Comment:
Removing parameters will also remove multiple test cases, right?

Reply:
Yeah, this was intentional. I felt the overhead of having parameterized tests was not worth the effort of testing each edge case here.

Reply:
So we test the success case and failed case only, right?